externals/kyuubi-spark-sql-engine/pom.xml

```diff
@@ -72,6 +72,13 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.spark</groupId>
+            <artifactId>spark-core_${scala.binary.version}</artifactId>
+            <type>test-jar</type>
```

Comment:
we can get rid of this dependencies if we fork `org.apache.spark.benchmark.Benchmark` from Spark

Reply:
The test-jar of `spark-core` must be included, as the `org.apache.spark.benchmark.Benchmark` class is part of it. The tests cannot be run without it.